tests/acceptance/bootstrap/FilesVersionsContext.php

```diff
@@ -559,19 +581,26 @@ public function listVersionFolder(
 	}
 
 	/**
-	 * This will run before EVERY scenario.
-	 * It will set the properties for this object.
-	 *
-	 * @BeforeScenario
+	 * @When user :user gets the number of versions of file :file from space :space
+	 * @When user :user tries to get the number of versions of file :file from space :space
 	 *
-	 * @param BeforeScenarioScope $scope
+	 * @param string $user
+	 * @param string $file
+	 * @param string $space
 	 *
 	 * @return void
 	 */
-	public function before(BeforeScenarioScope $scope):void {
-		// Get the environment
-		$environment = $scope->getEnvironment();
-		// Get all the contexts you need in this context
-		$this->featureContext = BehatHelper::getContext($scope, $environment, 'FeatureContext');
+	public function userGetsTheNumberOfVersionsOfFileInsideSpace(string $user, string $file, string $space): void {
+		$fileId = $this->spacesContext->getFileId($user, $space, $file);
+		$response = $this->featureContext->makeDavRequest(
+			$user,
+			"PROPFIND",
+			"/meta/$fileId/v",
+			null,
+			null,
+			null,
+			'2'
+		);
```

Comment:
I think, this args are not in correct order, what is `2`?

Reply:
it was in the correct order before but maybe some1 PR could bought change not sure